### ISSUE TITLE
​Improving the block selection button

### ISSUE THREAD

**Issue by EnterMan123** [toeverything/blocksuite#7903]
### Description

This is a suggestion feedback: 
Improve the button appearance style for selecting blocks

It's too small and has unnecessary animation effects. I'm often distracted from my limited attention by the shrinking animation effect. Additionally, it can be mistaken for something other than a button, perhaps its appearance could be improved to emphasize that it actually has more functionality.

![2](https://github.com/toeverything/AFFiNE/assets/107846196/a967f894-32c4-44f4-8893-1ccf2836669f)

gif 👇
![3](https://github.com/toeverything/AFFiNE/assets/107846196/973a9590-fdda-4438-beff-756eaf13d769)
![5](https://github.com/toeverything/AFFiNE/assets/107846196/995ff1fb-98a4-4d99-9a9d-4e63d5d97a80)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by hwangdev97** [comment]
Yes! I'm also not very satisfied with our current drag handle. We need some time to prepare a better drag handle. Thank you for your feedback.
